schema/site_metadata.json

```diff
@@ -92,6 +92,11 @@
           "description": "Source repository where the UDMI site model is stored",
           "type": "string",
           "examples": ["https://github.com/faucetsdn/udmi_site_model", "git@github.com:faucetsdn/udmi_site_model.git"]
+        },
+        "sheet": {
+          "description": "Link to a spreadsheet to observe real-time output from any server-side tool",
```

Comment:
it's really "any tool run" -- not necessarily just server side... e.g. if I run it at the CLI I would expect the same thing to happen.